### ISSUE TITLE
How to add a key when using wakid emacs?

### ISSUE THREAD

**Issue by onekk** [darkstego/wakib-keys#6]
I have a relatively simple task:

I want to use this package, that define a keymap binding.

```
(use-package numpydoc
  :ensure t
  :bind (:map python-mode-map
              ("C-c C-n" . numpydoc-generate)))
```

I was thinking that it will suffice to change the "C-c C-n" thing to  "C-d C-n" to make it works with wakib-keys.

But is not working.

I have searched around how this task is made in wakib code but even this code:

```
(use-package numpydoc
  :ensure t
  :config
  (define-key python-mode-map (kbd "C-c C-n") 'numpydoc-generate)
  )

```

seems not work.

I'm missing something.

I'm using ivy and cousel

TIA and Regards

Carlo D.

**Comment by darkstego** [comment]
Hi, sorry for the late reply. I just checked the second code snippet on my system and it works fine. Sometimes the bind option of use-package won't work because it uses `bind-keymap` and that doesn't work with the Wakib keymaps.

The second snippet is how I make things work usually. Can you tell me what happens when you run `describe-key`, so run `C-d k C-d C-n`. In my case it shows that key is mapped to `numpydoc-generate`.

**Comment by onekk** [comment]
Ok some more test:

I've used: 
`(define-key python-mode-map (kbd "C-c C-n") 'numpydoc-generate)`

It will appear correctly in the command list. maybe a flaws that has been corrected recently, because I have done many tests and stopped and restarted Emacs in between so I can't guess why now it is working. 

To resume mapping done by wakib-keys is transparent, every command added as `C-c` wil be translated in `C-d`?

Sorry for the noise.

Thank you.
Carlo D.
